nservicebus/tools/migrate-to-native-delivery.md

```diff
@@ -209,19 +207,17 @@ Use `SELECT * FROM TimeoutsMigration_State` to list all performed migrations. Fo
 
 ### NHibernate persistence
 
- - Delete all the entries from `StagedTimeoutEntity`, that contains the copy of migrated timeouts
- - Delete the table from `MigrationsEntity` that contains a row for every performed migration
+ - Delete the `StagedTimeoutEntity` table. This table contains copies of migrated timeouts.
```

Comment:
@lailabougria @danielmarbach I went ahead and changed this line to instruct the user to delete the entire table. Is that correct? I wasn't entirely sure from the previous conversation thread.

Reply:
Yes, as part of the cleanup after the migration, it does make sense to delete it.